decidim-core/lib/decidim/core/seeds.rb

```diff
@@ -13,6 +13,8 @@ def call
 
         reset_column_information
 
+        ActiveJob::Base.queue_adapter = :inline
```

Comment:
We need to set the queue adapter to inline, in order to actually seed the notifications area (as we fire events that needs to be processed)